### ISSUE TITLE
Add social media cards

### ISSUE THREAD

**Issue by plexus** [lambdaisland/gaiwan_co#6]
Especially for Twitter, so we get a nice preview when people share the link.

**Comment by oxalorg** [comment]
Yes that seems important, we can do it using: https://metatags.io/

**Comment by oxalorg** [comment]
Do we have a Gaiwan logo? or we can just use the font as the wordmark logo (it looks clean and nice):

![image](https://user-images.githubusercontent.com/4194289/100334275-92220f80-2ff9-11eb-8907-23f9cdc49433.png)

**Comment by plexus** [comment]
Yes, the typeface is the logo for Gaiwan. Felipe used Traja Sans for this, but that's an Adobe font which limits what we can do with it. Instead I'd like to suggest switching to Cinzel. It has more pronounced serifs, but has a similar "roman inscription" feel, and it's available under an open font license.

![2020-11-27-081245_1372x515_scrot](https://user-images.githubusercontent.com/32212/100421313-6b5eea00-3088-11eb-83c4-2f1d140aeff0.png)

**Comment by oxalorg** [comment]
100% agree, lets switch to Cinzel from now onwards 👍

**Comment by loopified** [comment]
I think Cinzel is a beautiful typeface, but when selecting Trajan Sans I was looking for a typeface that looked both humanist and eastern. The fact that it was inspired by Roman letters ended up being just an accident, as without the serifs it is barely recognizable as one. To me it looks like it is a marriage between pencil strokes and vectors, or like a sharp bamboo tip. In this sense, my opinion is that Cinzel isn't a nice alternative.

I'm not against finding an alternative and have tried to look for one, but so far I believe what Arne described on Discord is the best route. Using Trajan Sans for the logo (and therefore using it in SVG), which requires only the desktop license for one weight (and it also covers usage in print and other mediums), while choosing one or two Open Source typefaces to use for content.
